source/index.txt

```diff
@@ -152,6 +152,6 @@ Further Reading
    mongostat </mongostat>
    mongotop </mongotop>
    mongofiles </mongofiles>
-   /installation/installation
-   /logs
-   /release-notes/database-tools-changelog
+   Installation </installation/installation>
+   Logs </logs>
+   100.10.0 Changelogs </release-notes/database-tools-changelog>
```

Comment:
This is likely out of scope, but one thing I noticed is that we will have to manually update this ToC entry in `index.txt` when adding new release notes. It looks like we already manually move the contents of this page to a standalone file with the version number, so I don't see it being a big issue.

I wonder if there's a way to apply some kind of variable and replace functionality to this entry from the actual `/release-notes/database-tools-changelog` file. LMK if you have any thoughts, but maybe this could be a future improvement unrelated to this task.

Reply:
Thanks for this consideration - I'll bring this to @sarahemlin 's attention when she gets back!